Added tests for classifier

komainu8 opened the pull request:
Dear, developers.

Currently, classifier is not tested.
So, I added tests for classifier.

Commit:
Added tests for classifier

rimms commented:
Thanks.

`TestGetLabels` is failed.
```
--- FAIL: TestGetLabels (0.02s)
	classifier_test.go:54: got invalid get_labels response
```

rimms commented:
I found the cause of the above test failure.

We will fix this. Thanks.
https://github.com/jubatus/jubatus-go-client/issues/7

Commit:
Fix expected value type.

A change expected return value type "map[string]int64" to "map[string]uint64".
Because I've fixed return value type of GetLabels by "Issues #7".

rimms commented:
I tested this patch in develop branch. Tank you! 👍 

```
$ go test test/classifier_test.go -v
=== RUN TestSetLabel
--- PASS: TestSetLabel (0.01s)
=== RUN TestGetLabels
--- PASS: TestGetLabels (0.02s)
=== RUN TestDeleteLabel
--- PASS: TestDeleteLabel (0.01s)
=== RUN TestTrain
--- PASS: TestTrain (0.01s)
=== RUN TestClassify
--- PASS: TestClassify (0.01s)
=== RUN TestSave
--- PASS: TestSave (0.01s)
=== RUN TestLoad
--- PASS: TestLoad (0.01s)
=== RUN TestClear
--- PASS: TestClear (0.01s)
=== RUN TestGetStatus
--- PASS: TestGetStatus (0.01s)
PASS
ok  	command-line-arguments	0.124s
```
